app/models/UpdateCase.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import models.DocumentTypes.DocumentType
+import play.api.libs.json._
+import play.api.libs.functional.syntax._
+
+case class UpdateCase(caseId: String,
+                      documentsSupplied: Seq[DocumentType],
+                      supportingDocuments: Seq[SupportingDocument],
```

Comment:
the EIS API doc doesn't mention the supporting documents at all, but we collect them on the frontend, will we need this?